catalog/app/testing/RequestTracker.js

```diff
@@ -1,70 +0,0 @@
-import invariant from 'invariant';
```

Comment:
What became of the logic in this file?

Reply:
`fetch-mock` package provides similar functionality